endpoints/auction.go

```diff
@@ -269,8 +262,21 @@ func isUndimensionedBanner(bid *pbs.PBSBid) bool {
 	return bid.CreativeMediaType == "banner" && (bid.Height == 0 || bid.Width == 0)
 }
 
-func isBidIDAndCodeEqual(adunit *pbs.PBSAdUnit, bid *pbs.PBSBid) bool {
-	return adunit.BidID == bid.BidID && adunit.Code == bid.AdUnitCode
+func copyBannerDimensions(adunit *pbs.PBSAdUnit, bid *pbs.PBSBid, finalValidBids []*pbs.PBSBid, finalBidCounter *int) bool {
```

Comment:
May I ask what's the reason behind using pointers here for `finalValidBids` and `finalBidCounter`?

Reply:
`finalValidBids` was declared in line 243 as a dynamic array of pointers, therefore, I'm passing it as a parameter to `copyBannerDimensions` as such:
```
243     finalValidBids := make([]*pbs.PBSBid, len(bids))
   .
   .
265 func copyBannerDimensions(adunit *pbs.PBSAdUnit, bid *pbs.PBSBid, finalValidBids []*pbs.PBSBid, finalBidCounter *int) bool {
```
On the other hand `finalBidCounter` serves as a counter of the for loop in line 246 and gets modified inside `copyBannerDimensions` (line 272), therefore it needs to be passed as a reference